### PR TITLE
update Air Gap Installation

### PR DESCRIPTION
**Pull request by xiaoluhong**
https://github.com/longhorn/longhorn/issues/1641

### CODE REVIEW

File: content/docs/1.0.1/advanced-resources/deploy/airgap.md [hunk 1]
```diff
@@ -180,53 +180,86 @@ weight: 2
...
 
-## Using Helm Chart
 1. Clone longhorn repo
 
     `git clone https://github.com/longhorn/longhorn.git`
 
 2. In `chart/values.yaml`
 
-* Specify Longhorn images:
+    * Specify registryUrl,registryUser,registryPasswd, or registrySecret:
```
Comment (yasker) [inline content/docs/1.0.1/advanced-resources/deploy/airgap.md:199]: leave space after `,`. 

```
registryUrl, registryUser, registryPasswd, or registrySecret:
```

File: content/docs/1.0.1/advanced-resources/deploy/airgap.md [hunk 2]
```diff
@@ -180,53 +180,86 @@ weight: 2
...
    `kubectl apply -f longhorn.yaml`
+## Using Helm Chart
 
+In v1.0.2, Longhorn automatically adds <REGISTRY_URL> prefix to images. You simply need to set the registryUrl parameters to pull images from your private registry.
+
+> **Note:** Once you set registryUrl to your private registry, Longhorn tries to pull images from the registry exclusively. Make sure all Longhorn components' images are in the registry otherwise Longhorn will fail to pull images.
+
+### Use default image name
+
+If you keep the images' name as recommended [here](./#recommendation), you only need to do the flowing steps:
```
Comment (yasker) [inline content/docs/1.0.1/advanced-resources/deploy/airgap.md:191]: `images' names`

`following`

File: content/docs/1.0.1/advanced-resources/deploy/airgap.md [hunk 3]
```diff
@@ -180,53 +180,86 @@ weight: 2
...
+### Use custom image name
 
-    ```
-    csi:
-      attacherImage: <REGISTRY_URL>/csi-attacher:<CSI_ATTACHER_IMAGE_TAG>
-      provisionerImage: <REGISTRY_URL>/csi-provisioner:<CSI_PROVISIONER_IMAGE_TAG>
-      driverRegistrarImage: <REGISTRY_URL>/csi-node-driver-registrar:<CSI_NODE_DRIVER_REGISTRAR_IMAGE_TAG>
-      resizerImage: <REGISTRY_URL>/csi-resizer:<CSI_RESIZER_IMAGE_TAG>
-    ```
+If you want to use custom images' name, you can use the following steps:
```
Comment (yasker) [inline content/docs/1.0.1/advanced-resources/deploy/airgap.md:213]: `names`

Comment (xiaoluhong) [reply content/docs/1.0.1/advanced-resources/deploy/airgap.md]: Done. Please review, thank you.

File: content/docs/1.0.1/advanced-resources/deploy/airgap.md (hunk 1 above)
Comment (xiaoluhong) [reply content/docs/1.0.1/advanced-resources/deploy/airgap.md]: Done. Please review, thank you.

File: content/docs/1.0.1/advanced-resources/deploy/airgap.md (hunk 2 above)
Comment (xiaoluhong) [reply content/docs/1.0.1/advanced-resources/deploy/airgap.md]: Done. Please review, thank you.

**Comment by yasker** [comment]
Thanks @catherineluse !

**Pull request merged**